tests/testthat/test-find_groups.R

```diff
@@ -1,13 +1,10 @@
-context("find_groups")
-
 test_that("find_groups() success case", {
-
-  options('meetupr.use_oauth' = FALSE)
+  withr::local_options(list(meetupr.use_oauth = FALSE))
   set_api_key("R-Ladies FTW!")
 
   meetup_groups <- with_mock(
     `httr::GET` = function(url, query, ...) {
-      load(here::here("tests/testdata/httr_get_find_groups.rda"))
+      load(test_path("testdata/httr_get_find_groups.rda"))
```

Comment:
`testthat::test_path()` is a more robust way to build this path. `here::here()` isn't going to work in all settings. Plus this switch removes the dependency on here.

Reply:
I also moved the `.rda` files to put them in a place that's easy to access with `test_path()`.